about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Page</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <script
+      src="https://kit.fontawesome.com/e520ead38a.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="topnav">
+        <i class="fa-brands fa-facebook-f social-icon"></i>
+        <i class="fa-brands fa-twitter social-icon"></i>
+        <ul class="topnav-list">
+          <li class="top-links">English</li>
+          <li class="top-links">MyPage</li>
+          <li class="top-links">Logout</li>
+        </ul>
+      </div>
+      <div class="main-nav">
+        <img src="./imgs/logoimg.jpg" class="logo" alt="ICFP Logo">
```

Comment:
- [ ] Dear @Afsheen96, for better UX and semantic reasons, The logo better be a link to the homepage. It is conventional for the user to click on a page logo to navigate to the homepage. Please consider making your logo a hyperlink.

Reply:
have applied the changings @TadesseDev